pkg/clients/cloudwatch/client.go

```diff
@@ -1,4 +1,4 @@
-package apicloudwatch
+package cloudwatch
```

Comment:
To avoid the `cloudwatch_client` alias it might be good to keep `api` as a prefix or suffix, wdyt?

Reply:
I would like to leave it as is because the `cloudwatch_client` alias is just temporary for this change. After the next step the only cloudwatch references should come from this package or the models package.

Reply:
Makes sense!